designsafe/static/scripts/data-depot/components/data-depot-nav/data-depot-nav.component.html

```diff
@@ -37,5 +37,7 @@
     <div>
         <a href="/rw/user-guide/data-depot" target="_blank"><i class="fa fa-info-circle"></i> Learn About the <br/>Data Depot</a>
     </div>
-
+    <div>
+      <a href="https://www.designsafe-ci.org/faq/">FAQ</a>
```

Comment:
Can you turn this into a slug instead of an absolute url?
```suggestion
      <a href="/faq">FAQ</a>
```